benches/iterators.rs

```diff
@@ -129,7 +129,7 @@ fn bytes_iter_prev(bench: &mut Bencher) {
     let itr_src = r.bytes_at(r.len_bytes());
     let mut itr = itr_src.clone();
     bench.iter(|| {
-        if let None = itr.prev() {
+        if itr.prev().is_none() {
```

Comment:
Definitely like this change!  I didn't actually realize that method existed.  Thanks!